### PR TITLE
Abstract page layout

### PR DESCRIPTION
**Pull request by shinyichen**
### Added widgets
Screen Shot 2021-09-14 at 12 38 33 PM

### Responsive nav menu
Screen Shot 2021-09-14 at 12 38 13 PM

### Menu dropdown
Screen Shot 2021-09-14 at 12 40 04 PM

### CODE REVIEW

File: src/components/AbstractSideNav/AbstractSideNav.tsx [hunk 1]
```diff
@@ -14,41 +17,121 @@ export interface IAbstractSideNavProps extends HTMLAttributes<HTMLDivElement> {
...
+        label: itemElements[index],
+        domId: `absNav-${item.name}`,
+      };
+    });
+    const currentItem = navigation.find((item) => item.href === subPage);
+    const Icon = currentItem.icon || DocumentIcon;
+    const count = getCount(currentItem.href, doc);
+    const showCount = count > 0 && currentItem.href !== Routes.SIMILAR;
+    const label = (
+      <div className="text-gray-900' : 'text-gray-600 group flex items-center mt-5 px-3 py-2 text-left hover:text-gray-900 text-sm font-medium bg-gray-100 hover:bg-gray-50 rounded-md">
```
Comment (thostetler) [inline src/components/AbstractSideNav/AbstractSideNav.tsx:68]: looks like something is up with this `className` param

File: src/components/AbstractSideNav/AbstractSideNav.tsx [hunk 2]
```diff
@@ -14,41 +17,121 @@ export interface IAbstractSideNavProps extends HTMLAttributes<HTMLDivElement> {
...
+        <Icon className="flex-shrink-0 mr-3 w-6 h-6 text-gray-500" aria-hidden="true" />
+        <span className="flex-1 truncate">{currentItem.name}</span>
+        {showCount ? (
+          <span className={'bg-white inline-block ml-3 px-3 py-0.5 text-xs rounded-full'}>{count}</span>
```
Comment (thostetler) [inline src/components/AbstractSideNav/AbstractSideNav.tsx:72]: Probably identical behavior, but should switch this to be a string here for `className`

File: src/pages/abs/[id]/abstract.tsx [hunk 3]
```diff
@@ -14,6 +14,8 @@ export interface IAbstractPageProps {
 const AbstractPage: NextPage<IAbstractPageProps> = (props: IAbstractPageProps) => {
   const { doc, error } = props;
 
+  console.log(doc);
```
Comment (thostetler) [inline src/pages/abs/[id]/abstract.tsx:17]: remove console log

**Review by thostetler — CHANGES_REQUESTED** [review]
LGTM, just a couple small things to address I think.  Nice work!

File: src/components/AbstractSideNav/AbstractSideNav.tsx (hunk 1 above)
Comment (shinyichen) [reply src/components/AbstractSideNav/AbstractSideNav.tsx]: Fixed

File: src/pages/abs/[id]/abstract.tsx (hunk 3 above)
Comment (shinyichen) [reply src/pages/abs/[id]/abstract.tsx]: removed log

File: src/components/AbstractSideNav/AbstractSideNav.tsx (hunk 2 above)
Comment (shinyichen) [reply src/components/AbstractSideNav/AbstractSideNav.tsx]: Changed to string

**Pull request merged**